src/api/base.ts

```diff
@@ -16,7 +16,7 @@ const LOG_LOCAL = false;
 const LOGGER = new Logger(LIB_NAME, process.env.DEBUG === 'true' || LOG_LOCAL);
 
 export default class BaseAPI implements BaseClass {
-	protected client: RequesterClass;
+	protected client?: RequesterClass;
 	protected config: WAConfigType;
 
 	constructor(config: WAConfigType, HttpsClient?: RequesterClass) {
```

Comment:
@rashedtalukder said offline that client should not be optional. That's easy enough here but triggers a cascading issue because the webhooks constructor does not receive a client object.